oct_converter/readers/img.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, filepath):
         if not self.filepath.exists():
             raise FileNotFoundError(self.filepath)
 
-    def read_oct_volume(self, interlaced=False):
+    def read_oct_volume(self, rows=1024, cols=512, interlaced=False):
         """Reads OCT data.
```

Comment:
Could these two new arguments be added to the Args section of the function comment string?

Reply:
I've updated the Args section with the comments as suggested. Kindly take a look at it and see if its sufficient.